modules/nw-udn-best-practices.adoc

```diff
@@ -31,3 +31,14 @@ Before setting up a `UserDefinedNetwork` (UDN) resource, users should consider t
 
 * The cluster subnet and services CIDR for a UDN cannot overlap with the default cluster subnet CIDR. OVN-Kubernetes network plugin uses `100.64.0.0/16` as the default network's join subnet, you must not use that value to configure a UDN `joinSubnets` field. If the default address values are used anywhere in the cluster's network you must override it by setting the `joinSubnets` field. For more information, see "Additional configuration details for a UserDefinedNetworks CR".
 
+* {product-title} administrators should be aware that empty `matchLabel` selectors results in selecting all namespaces of the cluster.
+
+* For primary networks, the namespace used for the `UserDefinedNetwork` CR must include the `k8s.ovn.org/primary-user-defined-network` label. This label cannot be updated, and can only be added when the namespace is created. The following conditions apply with the `k8s.ovn.org/primary-user-defined-network` namespace label:
+
+** If the namespace is missing the `k8s.ovn.org/primary-user-defined-network` label and a pod is created, the pod attaches itself to the default network.
+
+** If the namespace is missing the `k8s.ovn.org/primary-user-defined-network` label and a primary UDN CR is created that matches the namespace, the UDN reports an error status and will not be generated.
```

Comment:
UDN reports an error status and the NAD will not be generated

or

UDN reports an error status and the network will not be created